Source/JavaScriptCore/runtime/FileBasedFuzzerAgent.cpp

```diff
@@ -55,14 +55,9 @@ SpeculatedType FileBasedFuzzerAgent::getPredictionInternal(CodeBlock* codeBlock,
 
     // FIXME: the output of codeBlock->expressionRangeForBytecodeIndex() allows for some of
     // these opcodes to have predictions, but not all instances can be reliably targeted.
-    case op_bitand: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
-    case op_bitor: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
-    case op_bitxor: // partially broken https://bugs.webkit.org/show_bug.cgi?id=203604
```

Comment:
Looks like https://bugs.webkit.org/show_bug.cgi?id=203604 is no longer relevant.  Let's close it as Config Changed.

Reply:
Done